wagtail/admin/tests/pages/test_explorer_view.py

```diff
@@ -717,47 +717,3 @@ def test_breadcrumb_with_no_user_permissions(self):
             response,
             """<li class="home breadcrumb-item"><a class="breadcrumb-link" href="/admin/pages/4/" class="icon icon-home text-replace">Home</a></li>""",
         )
-
-
-@override_settings(WAGTAIL_I18N_ENABLED=True)
-class TestLocaleSelector(TestCase, WagtailTestUtils):
```

Comment:
Rather than remove those tests, I’m wondering if we should change them to check the contents of the side panels? Same for the `test_privacy` ones. What do you think?

Reply:
We might not need them here as we discussed in the meeting. Marking this as resolved.